src/downshift.js

```diff
@@ -148,7 +153,7 @@ class Downshift extends Component {
   }
 
   getItemNodeFromIndex = index => {
-    return document.getElementById(this.getItemId(index))
+    return this._rootNode.querySelector(`#${this.getItemId(index)}`)
```

Comment:
This won't work for folks rendering the menu outside the autocomplete root node I'm afraid. Need to set this as the eventTarget. Due to that fact, maybe eventTarget is not quite the right name... Maybe simply "window" ?

Reply:
We could also consider "context", since the new docs say that the type can be `Element`/`document`/`window`.

Reply:
I'd be fine with that too, tough I hesitate because I worry folks might confuse that with React's context feature 🤔

Reply:
Agreed. I realized right after I typed it that React also has "context".

Looked at the [thesaurus entry for "context"](http://www.thesaurus.com/browse/context). I'll throw out "environment".

I'll also suggest "renderContext".

Reply:
I'm good with either of those. Like the docs said, most people shouldn't be using this prop anyway.

Reply:
"environment" works well for other use cases like React Native (https://github.com/paypal/downshift/issues/185)